buildenv/jenkins/variables/defaults.yml

```diff
@@ -545,17 +545,13 @@ x86-64_mac:
     8: 'macosx-x86_64-normal-server-release'
     11: 'macosx-x86_64-normal-server-release'
   extra_configure_options:
-    8: '--with-xcode-path=/Users/jenkins/Xcode4/Xcode.app --with-openj9-cc=/Users/jenkins/Xcode7/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang --with-openj9-cxx=/Users/jenkins/Xcode7/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ --with-openj9-developer-dir=/Users/jenkins/Xcode7/Xcode.app/Contents/Developer'
+    8: '--with-toolchain-type=clang'
   freemarker: '/Users/jenkins/freemarker.jar'
   openjdk_reference_repo: '/Users/jenkins/openjdk_cache'
   node_labels:
-    build:
-      all: 'ci.role.build && hw.arch.x86 && sw.os.osx.10_13'
-      8: 'ci.role.build && hw.arch.x86 && sw.os.osx.10_11'
+    build: 'ci.role.build && hw.arch.x86 && sw.os.osx.10_14'
   build_env:
-    vars:
-      all: 'OPENJ9_JAVA_OPTIONS=-Xdump:system+java:events=systhrow,filter=java/lang/ClassCastException,request=exclusive+prepwalk+preempt'
-      8: 'MACOSX_DEPLOYMENT_TARGET=10.9.0 SDKPATH=/Users/jenkins/Xcode4/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.8.sdk'
+    vars: 'MACOSX_DEPLOYMENT_TARGET=10.10.0 OPENJ9_JAVA_OPTIONS=-Xdump:system+java:events=systhrow,filter=java/lang/ClassCastException,request=exclusive+prepwalk+preempt'
```

Comment:
Why is the deployment target updated from 10.9.0 to 10.10.0? I think it should just be removed; I don't think it has any effect anyway - build and configure scripts appear to ignore it.

Reply:
Its not used by xcode/clang to support older releases?

Reply:
Even if it is used by the compiler, the value supplied here is ignored; see e.g. https://github.com/ibmruntimes/openj9-openjdk-jdk11/blob/openj9/closed/autoconf/custom-spec.gmk.in#L109.

Reply:
Right and set in https://github.com/ibmruntimes/openj9-openjdk-jdk11/blob/22c634c40143ed77119efa3800a8a9f1a726d19f/make/autoconf/flags.m4#L119 to 10.9.0, understood.  So back to you @olvap377 to remove `MACOSX_DEPLOYMENT_TARGET=10.10.0 `

Reply:
Removed